index.html

```diff
@@ -32,8 +32,8 @@
     <div class="column">
     </div>
     <div class="column">
+      <div id="example"><script>include("articles/example.md", "example")</script></div>
```

Comment:
Please rename everything to the actual article name

Reply:
Renamed to parachutes